### DISCUSSION TITLE
Why is this free?

### DISCUSSION THREAD

**Discussion by bn-l** [loophole/cli#114] [Q&A]
Hi there, how come this is free at the moment? Do you have plans to charge in the future?

**Comment by 0x7f** [comment]
Hi Ben, thanks for reaching out. Yes, right now the service is free as it is in beta phase. We are not sure yet where to slice the product in the end. So which features will remain free and which will be paid. Open for input here :) For now and for the upcoming months, the product stays free.

**Reply by bn-l** [reply]
Hi Max, this might be a stupid question, but can loophole technically see my traffic (even considering the connection is TLS'd)? Sorry I don't have a lot of knowledge / exp. with networking.

Re. pricing, if you want my advice: keep custom domains free. I had a look at many of these services on an "awesome" type list. Yours was the only one that had custom domains and TLS free. I would keep this but set an abuse limit on the free tier.

I would then charge $2 a month for full speed and $5 for this plus some analytics as a pro tier. Just my thoughts. These prices are the market rate so it seems it wouldn't be worthwhile to differentiate on price.

**Reply by 0x7f** [reply]
Hi Ben, thanks again for your input. Really appreciate it :)

Re security: we can not see any of your traffic. It is sent encrypted to your client and only you can decrypt it. We are in the process of writing a blog post on how this works in detail. Will let you know when we post it.

Re pricing: thanks for your input. Keeping TLS is a given on our end. Our tunnels will always be fully encrypted. Custom domains is sth we considered for a premium model. We might rethink it based on your input 😊

**Reply by jmeri** [reply]
I second the motion of keeping custom domains free. :-)
My need for this kind of service is occasional, but loophole seems to do all I need. And it does it well.

**Reply by msanaullahsahar** [reply]
I third keeping the custom domain free model but only for 2 or 3 tunnel. Then you can charge if someone need more custome domain tunnels.
